content/observability-slurm/README.md

```diff
@@ -0,0 +1,693 @@
+# Installing observability stack on SLURM
+
+## Automatic
+The easiest way of creating SLURM cluster with observability stack enabled is to use our Terraform/Ansible script. Make sure to enable observability in your values:
+```hcl
+enable_observability = true
+```
+
+## Manual steps
+
+### Prerequisites
+- SLURM Cluster: A functioning SLURM cluster with head nodes and compute nodes.
+- Administrative Access: Root or sudo privileges on all nodes.
+- Internet Access: Nodes must be able to download packages from the internet.
+- NVIDIA GPUs: Compute nodes with NVIDIA GPUs and drivers properly installed.
+- Operating System: Ubuntu 20.04 or newer
+- Firewall: Ability to configure firewall rules for required ports
+
+### Security Considerations
```

Comment:
Are these inbound rules only required for head node or all VMs in VPC? We allow all egress traffic by default, but for security should only be exposing necessary inbound ports